QuickTests/Helpers/QCKSpecRunner.h

```diff
@@ -10,7 +11,7 @@
  @param specClass The class of the spec to be run.
  @return An XCTestRun instance that contains information such as the number of failures, etc.
  */
-extern XCTestRun *qck_runSpec(Class specClass);
+QUICK_EXPORT XCTestRun *qck_runSpec(Class specClass);
```

Comment:
Ops(again)...  Yes, you are right, it was lack of attention, my bad. I'm going to remove.